docs/user-guide.md

```diff
@@ -24,34 +25,50 @@ This guide provides detailed instructions on how to configure your Liberty proje
 ## Before you begin
 
 ### Software requirements
-
-- Java 17 is required by Liberty Tools for Visual Studio Code.
+<!-- Match the VSCode and Java version with the README.md -->
+- **Visual Studio Code 1.78.0** or later.
+- **Java 17** or later is required by Liberty Tools for Visual Studio Code. To point Visual Studio Code to a specific runtime, see [Settings](#settings) for more information.
 
 ### Application requirements
 
 - Define a Liberty `server.xml` configuration file in the `src/main/liberty/config` location.
-- Configure the [Liberty Maven Plugin](https://github.com/OpenLiberty/ci.maven#configuration) or [Liberty Gradle Plugin](https://github.com/OpenLiberty/ci.gradle#adding-the-plugin-to-the-build-script). We recommend using recent versions of the plugins as they include several important bug fixes.
+- [Configure the Liberty Maven Plugin](https://github.com/OpenLiberty/ci.maven#configuration) or [configure the Liberty Gradle Plugin](https://github.com/OpenLiberty/ci.gradle#adding-the-plugin-to-the-build-script). We recommend using the most recent versions of the plugins as they include several important bug fixes.
+
+    - [Liberty Maven Plugin latest release](https://github.com/OpenLiberty/ci.maven/releases/latest)
+
+    - [Liberty Gradle Plugin latest release](https://github.com/OpenLiberty/ci.gradle/releases/latest)
 
-  The following versions are recommended at minimum:
-  - Liberty Maven Plugin -> 3.7.1
-  - Liberty Gradle Plugin -> 3.5.1
+### Configure your Java
 
-### Settings
+Liberty Tools for Visual Studio Code requires Java 17 or later to ensure that the [Liberty Config Language Server](https://github.com/OpenLiberty/liberty-language-server), the [Eclipse Language Server for Jakarta EE](https://github.com/eclipse/lsp4jakarta), the [XML Language Server](https://github.com/eclipse/lemminx) and start properly. A toast message alerts you if any language server fails to run or if Visual Studio Code has trouble locating your Java.
```

Comment:
and the [XML Language Server](https://github.com/eclipse/lemminx) start properly.

Reply:
you might consider putting the language servers into a list for readability
```suggestion
Liberty Tools for Visual Studio Code requires Java 17 or later to ensure that the following underlying language servers start properly:
- [Liberty Config Language Server](https://github.com/OpenLiberty/liberty-language-server)
- [Eclipse Language Server for Jakarta EE](https://github.com/eclipse/lsp4jakarta)
- [XML Language Server](https://github.com/eclipse/lemminx) 

A toast message alerts you if any language server fails to run or if Visual Studio Code has trouble locating your Java.
```

not sure if "underlying" is an accurate characterization, edit as needed